importanize/formatters.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, unicode_literals
+
+import itertools
+import operator
+
+
+def get_normalized(i):
+    return map(operator.attrgetter('normalized'), i)
+
+
+class Formatter(object):
+    """Parent class for all formatters
+
+    Parameters
+    ----------
+    statement : ImportStatement object
+        This is the data-structure which store information about
+        the import statement that must be formatted
+    """
+    def __init__(self, statement):
+        self.statement = statement
+
+
+class IndentWithTabsFormatter(Formatter):
```

Comment:
trying to come up with better descriptive names for these. maybe this one - `GroupedFormatter` with `name='grouped'`

so some definitions:

* grouped - use of `()`